# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini
# Commit: 4739af17e757
# Full Hash: 4739af17e757724c0833a8564579833249389566
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-02-27 21:52:52
# Regressor Bug: 1920115
# File Overlap Count: 2
# Description:
#   Bug 1920115 - Allow high priority tasks to run before timers r=smaug
#   
#   In Gecko, unlike other runnables where the TaskController
#   picks one each at a time, the TimeoutManager allows multiple
#   timers to run sequentially without allowing other runnables
# ==============================================================================

diff -r 0773f669c498 -r 4739af17e757 testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini
--- a/testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini	Thu Feb 27 16:17:36 2025 +0000
+++ b/testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini	Thu Feb 27 16:17:36 2025 +0000
@@ -1,8 +1,6 @@
 [yield-priority-timers.any.worker.html]
+  # Currently worker doesn't use the TimeoutManager.
+  # This test needs to be investigated when `dom.workers.timeoutmanager` is enabled.
+  # Bug 1947484
   [yield() with timer tasks (inherit signal)]
     expected: FAIL
-
-
-[yield-priority-timers.any.html]
-  [yield() with timer tasks (inherit signal)]
-    expected: FAIL
